### ISSUE TITLE
Update Account Does Not Work From Dupe Logins

### ISSUE THREAD

**Issue by DanVanAtta** [triplea-game/triplea#4970]
### Problem
If logged in to lobby with a "dupe" login,  the 'update account' menu does not work and errors out.

### Repro

* Log in to lobby
* With a second game instance, log in to lobby. Notice the name has a " (1)" suffix
* Click "Account" > "Update Account". Notice, "no user found" error message is displayed

![Screenshot from 2019-08-14 19-07-04](https://user-images.githubusercontent.com/12397753/63068134-f2ee0d00-bec6-11e9-8f85-6db60e412c31.png)

### Details
In LobbyMenu.java:304, name is obtained from the local node:
```
final String userName = lobbyFrame.getLobbyClient().getMessengers().getLocalNode().getName();
```

In UserManager.java, we get and verify the name:

```
    final INode remote = MessageContext.getSender();
    if (!userName.equals(remote.getName())) {
      log.severe(
```

The node name does not correspond to the DB username in the case of multiple logins, which goes on to cause an error.

**Comment by OrdoFlammae** [comment]
Wouldn't the problem be that you are even allowed to log in twice? I cannot see why that's good.

**Comment by DanVanAtta** [comment]
Not a bad question/consideration. @ron-murhammer , @RoiEXLab , @prastle , what do you think if we disallow multiple connections to lobby and just the one login?

**Comment by RoiEXLab** [comment]
IMO this is a bad thing, because this would prevent people in the same network from playing together or against each other in the lobby.

**Comment by prastle** [comment]
We need a way to differentiate between multiple comps on the same network.  This has been a problem for years at triplea.  It is mainly used by trolls.  BUT!  That being said we do have families that play together online.  Example the Brother's Grimm comes to mind.  Is there anyway of using the mac to differentiate?

**Comment by prastle** [comment]
In general we should disallow multiple logins.

**Comment by Cernelius** [comment]
I'm against going as hard as disallow multiple connections (I guess from a same IP?) completely, but this should rather be discussed with the moderators (in bunker).

So just chirping in cause @prastle suggested me to, but I'm done here.

**Comment by OrdoFlammae** [comment]
I was actually specifically thinking of signing in twice with the same account (linked to email and username), not IP. Or is that the same thing for some reason. I feel like there's an element in this conversation that I'm missing. It seems like it should be an error to sign in with the same username.

**Comment by prastle** [comment]
It should be an error to sign in twice using different IP's same account as well or at the very minimum produce a "(1)" .  I think what you are missing in this conversation is our methods of security.  Perhaps we should move this discussion to the bunker as @Cernel suggested.

**Comment by OrdoFlammae** [comment]
@DanVanAtta took care of what I was talking about in #5079

**Comment by DanVanAtta** [comment]
We can open a new issue if there is still question whether we should reject duplicate logins rather than accepting and appending a "(1)" to the player name. The rejecting dupe logins is a significant update and worthy of a dedicated conversation thread.

**Comment by DanVanAtta** [comment]
Otherwise seems we've taken care of the issues originally presented, please re-open if there is more to discuss on those topics.

**Comment by DanVanAtta** [comment]
Original problem was not fixed, opening to track.

**Comment by prastle** [comment]
@DanVanAtta  My personal opinion is that duplicate logins at the same time should be completely disallowed.  If it is being handled by the computer "id" this should not be a problem.   This allows for multiple logins from the same household as discussed.  Can you explain further the problem?

**Comment by DanVanAtta** [comment]
@prastle see repro steps in OP and notice there is an error message when selecting the update account menu.

I'm inclined to agree that duplicate logins should not be allowed, but we need to think through the implications. It is certainly useful for development/testing. Otherwise thinking to when we see duplicate logins, it is when a user account is 'ghosted'. If we do not solve that problem, then a user could have a game crash, become ghosted, and then not be able to log back in.

**Comment by prastle** [comment]
@DanVanAtta  Very valid points.  I guess the ghosting needs to  be solved then or a 1 added by comp id.  dev testing I don't think matters since most of us have multiple ways to connect :)

**Comment by DanVanAtta** [comment]
@prastle how do you mean by multiple ways to connect? Are you referring to multiple devices at home?

**Comment by prastle** [comment]
Correct or multiple devices in a lot of places.  Thus why both comp id and service providers need to be part of the toolbox.

**Comment by DanVanAtta** [comment]
@prastle service providers? How do you mean exactly, ISP?

**Comment by DanVanAtta** [comment]
This is fixed in 2.0 where dupe logins are assigned the same name.
